libs/agno/agno/os/app.py

```diff
@@ -318,6 +328,51 @@ def _add_built_in_routes(self, app: FastAPI) -> None:
             self.interfaces.append(a2a_interface)
             self._add_router(app, a2a_interface.get_router())
 
+    def _check_duplicate_ids(
+        self,
+        items: Optional[List[Any]],
```

Comment:
This is a weird interface... you only need the `item_type` for a log? 

also `items` should be `entities`